doc/duckarrays.rst

```diff
@@ -33,4 +33,12 @@ will still cast to ``numpy`` arrays:
   * :py:meth:`Dataset.rolling` and :py:meth:`DataArray.rolling` (uses internal functions
     of ``numpy``)
   * :py:meth:`Dataset.interpolate_na` and :py:meth:`DataArray.interpolate_na` (uses
-    :py:func:`numpy.vectorize`)
\ No newline at end of file
+    :py:func:`numpy.vectorize`)
+
+
+Extensions using duck arrays
+----------------------------
+Here's a list of libraries extending ``xarray`` to make working with wrapped duck arrays
+easier:
+
+- `pint-xarray <https://github.com/xarray-contrib/pint-xarray>`_
```

Comment:
@jacobtomlinson do you want to list `cupy_xarray` here or is it too early to do that?

Reply:
I would say it's too early for that. But good to know where to put it.